src/components/atoms/NavigationPill.tsx

```diff
@@ -27,7 +25,7 @@ export default styled(NavigationPill)`
   font-family: "Open Sans", sans-serif;
```

Comment:
Font familiy is set in GlobalStyle and don't need to be set here.

Reply:
Fixed in local branch. Will push with another PR.